javascript/apps/taiga-e2e/src/e2e/project/kanban/kanban.cy.ts

```diff
@@ -205,4 +205,57 @@ describe('Kanban', () => {
       cy.get('tg-kanban-status').first().should('not.contain', statusName);
     });
   });
+
+  it('delete status moving stories', () => {
+    cy.getBySel('status-name').first().should('contain', 'New');
+    cy.get('tg-kanban-story').its('length').as('storiesCount');
+    cy.getBySel('status-options').first().click();
+    cy.getBySel('delete-status-btn').should('be.visible');
+    cy.getBySel('delete-status-btn').click();
+    cy.getBySel('submit-delete-status').click();
+
+    cy.getBySel('status-name').first().should('not.contain', 'New');
+    cy.getBySel('status-name').first().should('contain', 'Ready');
+    cy.get<number>('@storiesCount').then(function () {
```

Comment:
This assumes that there are only stories in the first status and that they are moved into the second (then first) status. Is this true forever?

Reply:
That's true as far as the default behaviour when we create a project is to have the 'new, in progress, ready, done' status created. Also, before all tests we are creating 1 story in the first status.